### DISCUSSION TITLE
Hotkey for "Not interested in this post"

### DISCUSSION THREAD

**Discussion by LTK1** [AprilSylph/XKit-Rewritten#1566] [Suggestion Box]
When posts are added to my dashboard based on tags I follow, the same posts usually shows up a bunch of times in the same place, even when I refresh the feed. I can open the menu and click "Not interested in this post" to prevent this; it will usually not show up again for a while. Could this option be bound to a hotkey so I can dismiss these posts more easily?

![image](https://github.com/user-attachments/assets/1006d28c-1989-4860-8ff8-da8a360df4f8)

**Comment by marcustyphoon** [comment]
You can turn those off entirely at the bottom of https://www.tumblr.com/settings/dashboard, I believe. Or do you want to keep them on, but not see certain ones?

**Reply by LTK1** [reply]
I want to keep them on, I just don't want to see them repeatedly. The same tagged post always shows up multiple times across dashboard refreshes if I don't click "Not interested in this post". I figured it might not be feasible to hide each such post after I've seen it so a hotkey for this function seemed like the next best thing.
